### PR TITLE
“publish test workflow dev”

### PR DESCRIPTION
**Pull request by Toby-Osborne**
“publish test workflow dev”
In all seriousness, I think its working now.  Note, that it will only publish it if you update the version number.  I might be able to find a fix for this but I do not currently have one.  Perhaps it doesn't need fixing.

### CODE REVIEW

File: .github/workflows/python-test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: python-test
+on: [push]
```
Comment (Jack17432) [inline .github/workflows/python-test.yml:2]: Can we have it also on pull request (just for the test workflow)

**Review by Jack17432 — CHANGES_REQUESTED** [review]
Just need to change one thing and then its all good :)

**Comment by Toby-Osborne** [comment]
Added the on pull request trigger to the testing workflow only

**Comment by Jack17432** [comment]
Looks good :)

**Pull request merged**